Makefile

```diff
@@ -18,16 +18,19 @@ cleanBuild:
 	make install
 
 db-gen:
-	cd packages/engine-server && yarn prisma generate 
+	cd packages/engine-server && yarn prisma generate
 
 start-local-registry:
 	yarn config set registry http://localhost:4873
 	npm set registry http://localhost:4873/
-	npx verdaccio -c ./bootstrap/data/verdaccio/config.yaml 
+	npx verdaccio -c ./bootstrap/data/verdaccio/config.yaml
 
 publish-local:
 	lerna publish from-package --ignore-scripts
 
 build-plugin:
 	dendron dev prep_plugin && rm package.json
-	dendron dev package_plugin
\ No newline at end of file
+	dendron dev package_plugin
+
+export-test:
```

Comment:
name is not very descriptive. would rename. eg. `setup-nextjs-test`

Reply:
[`ca81dfd` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/ca81dfd63062888a71517dce817f054d81e83a45)